lib/adapters/tool/testplane/runner/index.ts

```diff
@@ -1,9 +1,10 @@
 import _ from 'lodash';
```

Comment:
All testplane runners moved to testplane folder. They cannot be used for pwt (for example)